### PR TITLE
Unignore files for formatting (.fantomasignore) if the underlying Fantomas issue has been resolved

### PR DESCRIPTION
**Pull request by T-Gro**
### Fantomas limitation https://github.com/fsprojects/fantomas/issues/2264
This is resolved now
Was affecting src/FSharp.Core/SI.fs

### Fantomas limitations on signature files in FSharp.Core (https://github.com/fsprojects/fantomas/issues/2230)
This is resolved now
Was affecting src/FSharp.Core/array.fsi
Was affecting src/FSharp.Core/tasks.fsi
Was affecting src/FSharp.Core/seq.fsi

### CODE REVIEW

**Comment by nojaf** [comment]
Thanks for following up on this.

File: .fantomasignore [hunk 1]
```diff
@@ -109,8 +109,8 @@ src/FSharp.Core/Query.fs
 src/FSharp.Core/seqcore.fs
 
 # Fantomas limitation https://github.com/fsprojects/fantomas/issues/2264
-
-src/FSharp.Core/SI.fs
+# This should be resolved
```
Comment (psfinaki) [inline .fantomasignore:112]: Just don't forget to remove the commented lines for good

**Review by psfinaki — APPROVED** [review]

File: .fantomasignore [hunk 2]
```diff
@@ -141,7 +141,7 @@ src/FSharp.Core/resumable.fsi
...
 
 # Fantomas limitations on signature files in FSharp.Core (https://github.com/fsprojects/fantomas/issues/2230)
-
-src/FSharp.Core/array.fsi
-src/FSharp.Core/tasks.fsi
-src/FSharp.Core/seq.fsi
+# should be resolved
+# src/FSharp.Core/array.fsi
+# src/FSharp.Core/tasks.fsi
+# src/FSharp.Core/seq.fsi
```
Comment (T-Gro) [inline .fantomasignore:147]:
```suggestion
```

File: .fantomasignore [hunk 3]
```diff
@@ -141,7 +141,7 @@ src/FSharp.Core/resumable.fsi
 src/FSharp.Core/async.fsi
 
 # Fantomas limitations on signature files in FSharp.Core (https://github.com/fsprojects/fantomas/issues/2230)
-
-src/FSharp.Core/array.fsi
-src/FSharp.Core/tasks.fsi
-src/FSharp.Core/seq.fsi
+# should be resolved
```
Comment (T-Gro) [inline .fantomasignore:144]:
```suggestion
```

File: .fantomasignore [hunk 4]
```diff
@@ -141,7 +141,7 @@ src/FSharp.Core/resumable.fsi
 src/FSharp.Core/async.fsi
 
 # Fantomas limitations on signature files in FSharp.Core (https://github.com/fsprojects/fantomas/issues/2230)
```
Comment (T-Gro) [inline .fantomasignore:143]:
```suggestion
```

File: .fantomasignore [hunk 5]
```diff
@@ -109,8 +109,8 @@ src/FSharp.Core/Query.fs
 src/FSharp.Core/seqcore.fs
 
 # Fantomas limitation https://github.com/fsprojects/fantomas/issues/2264
-
-src/FSharp.Core/SI.fs
+# This should be resolved
+# src/FSharp.Core/SI.fs
```
Comment (T-Gro) [inline .fantomasignore:113]:
```suggestion
```

File: .fantomasignore (hunk 1 above)
Comment (T-Gro) [inline .fantomasignore:112]:
```suggestion
```

File: .fantomasignore [hunk 6]
```diff
@@ -109,8 +109,8 @@ src/FSharp.Core/Query.fs
 src/FSharp.Core/seqcore.fs
 
 # Fantomas limitation https://github.com/fsprojects/fantomas/issues/2264
```
Comment (T-Gro) [inline .fantomasignore:111]:
```suggestion
```

**Review by abonie — APPROVED** [review]

**Pull request merged**